python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/env python
+
+import asyncio
+import collections
+
+import numpy as np
+
+from lsst.ts import salobj
+from lsst.ts import scriptqueue
+
+import SALPY_ATMonochromator
+import SALPY_Electrometer
+import SALPY_FiberSpectrograph
+import SALPY_ATCamera
+import SALPY_ATSpectrograph
+import SALPY_ATArchiver
+import csv
+import datetime
+import os
+import pathlib
+import requests
+
+__all__ = ["CalSysTakeNarrowbandData"]
+
+def is_sequence(value):
```

Comment:
import from `python/lsst/ts/externalscripts/auxtel/calsys_takedata.py`

Reply:
I'm not sure what you're trying to say here.

Reply:
The method `is_sequence` is also defined in `python/lsst/ts/externalscripts/auxtel/calsys_takedata.py`. That means you should be able to do 

`from lsst.ts.externalscripts.auxtel.calsys_takedata import is_sequence`

same for `as_array` later.